doc/source/whatsnew/v3.0.0.rst

```diff
@@ -663,9 +663,9 @@ Interval
 
 Indexing
 ^^^^^^^^
+- Bug in :func:`pprint_thing` with ``quote_strings=True`` failing for strings with embedded single quotes (:issue:`60190`)
```

Comment:
`pprint_thing` is an internal function, the whatsnew is for users of pandas. We need to report on the impact users will notice. Something like ``Bug in printing :class:`Index` names would not escape single quotes``

Reply:
Changes have been made